app/chrony/README.md

```diff
@@ -2,21 +2,19 @@
 
 Basic chrony NTP client monitoring.
 
-* [Template App chrony active](Template_App_chrony_active.xml)
-
 This template is part of [RaBe's Zabbix template and helpers
 collection](https://github.com/radiorabe/rabe-zabbix).
 
-## Template
-
-### Items
-
-* Memory usage (rss) of "chronyd" processes (proc.mem[chronyd,chrony,,,rss])
-* Number of "chronyd" processes (proc.num[chronyd,chrony])
+## Template App chrony active
 
+### Items 
+* Memory usage ($5) of "$1" processes (`proc.mem[chronyd,chrony,,,rss]`)
```

Comment:
I made a gory exslt hack in b83a0d1dd0b74320e1afd82f9fc2a2cbbaa63ede. The code isn't nice and only works for up to 5 args but we can extend it if we need more.